### PR TITLE
Server skeleton

### PR DESCRIPTION
**Pull request by Denys-Bykov-0**

### CODE REVIEW

File: src/Net/Constants.cpp [hunk 1]
```diff
@@ -0,0 +1,16 @@
+#include "Constants.h"
```
Comment (AndriyFedorenko) [inline src/Net/Constants.cpp:1]: You don't need cpp file. It is enough to have only header file.

File: src/Net/Message.h [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+    QJsonDocument json_playload;
+};
+
+
+struct Request : public Message
+{
+	HTTPMethod method;
+	AuthSchemeType auth_scheme;
+    std::string auth_info;
+	QJsonDocument JWT_token_body;
```
Comment (AndriyFedorenko) [inline src/Net/Message.h:25]: If variable name in structure is lowercase than this rule should applied for all variables.

File: src/Server/Handlers/AuthorizedHandler.cpp [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+	return QJsonDocument::fromJson(raw_token_body);
+}
+
+void AuthorizedHandler::CheckAuthorization(const Net::Request& request)
+{
+	if (request.auth_scheme != Net::AUTH_SCHEME_TYPE_BEARER)
+		throw Net::UnauthorizedError("Unsupported authorization scheme");
+
+	try {
+		Poco::JWT::Signer signer("0123456789ABCDEF0123456789ABCDEF");
```
Comment (AndriyFedorenko) [inline src/Server/Handlers/AuthorizedHandler.cpp:53]: Please move it to constant

File: src/Server/Server.cpp [hunk 4]
```diff
@@ -0,0 +1,178 @@
...
+	}
+	catch (const Net::ServerError& ex) {
+		std::cout << ex.what() << "(URI: " << http_req.getURI() << ")" << "\n";
+		SendResponse(FormErrorResponse(ex), http_res);
+	}
+	catch (const std::exception& ex) {
+		std::cout << "Unhandled exception \"" << ex.what() << "\""
+		<< "(URI: " << http_req.getURI() << ")" << "\n";
+		SendResponse(FormErrorResponse(ex), http_res);
+	}
```
Comment (AndriyFedorenko) [inline src/Server/Server.cpp:48]: It would nice to have catch(...) for unhandled exception

**Review by MaksymTerekhovskyy — APPROVED** [review]
seems to be with no mistakes

**Review by AndriyFedorenko — APPROVED** [review]

**Pull request merged**